### PR TITLE
Resolved conflicts

### PR DESCRIPTION
**Pull request by Samuel325**
Resolved merge conflicts from this branch: https://github.com/P1Gaming/SkyCore/pull/77.

### CODE REVIEW

**Comment by Samuel325** [comment]
[Programming: Petting States](https://trello.com/c/o3p8Jy2x/991-programming-petting-states)

**Review by CaseyWalshWarder — COMMENTED** [review]
The jellies neediness (I think _currNumOfPets in Parameters) should decrease by 1 every x seconds. That setting should be editable in the inspector.

The animator should play the animation via its state machine, rather than using the animator.Play method. Otherwise, the animations will become more difficult to manage over time. If you don't have experience with the animator window, I can help after a meeting, and you should look up a tutorial for the animator window.
image

There's info in the flowchart about needing to have an empty hand, but I'm not sure whether they want the player to physically pet the jelly with the cursor or what that'd involve. We probably need clarification from design.

File: Assets/Tests/PlayMode/JelliesParametersTests.cs [hunk 1]
```diff
@@ -65,4 +65,17 @@ public IEnumerator TestIfJellyGetsHungrierAfter10Seconds()
...
+    {
+        CreateTestJelly(out GameObject testJelly, out Parameters parameters);
+        parameters.ReqAmountOfPetsPerState = new int[1];
+        parameters.ReqAmountOfPetsPerState[0] = 1;
+
+        parameters.IncreasePetCounter();
+
+        yield return null;
+        Assert.AreEqual(MoodStates.Slightly_Needy, parameters.MoodState);
+    }
```
Comment (CaseyWalshWarder) [inline Assets/Tests/PlayMode/JelliesParametersTests.cs:80]: We haven't been testing with unit tests when we review pull requests. Maybe we should discuss doing unit tests during a meeting.

**Comment by Samuel325** [comment]
Applied some of the changes according to the flowchart.

**Comment by CaseyWalshWarder** [comment]
The mood text doesn't update when it decrements the number of pets. 

The name of _petInterval is confusing because it's used for the time between when you can pet, whereas in a different script, it's the time between the number of pets decrements.

**Comment by Samuel325** [comment]
Refractured some code to make it a little clearer. Also fixed a bug with the mood state not decreasing.

**Review by CaseyWalshWarder — APPROVED** [review]
Looks good, nice job.

**Pull request merged**